src/cache/redis.rs

```diff
@@ -32,8 +33,12 @@ pub struct RedisCache {
 impl RedisCache {
     /// Create a new `RedisCache`.
     pub fn new(url: &str) -> Result<RedisCache> {
+        let mut parsed = Url::parse(url)?;
+        if parsed.password().is_some() {
```

Comment:
Could you please add a comment to explain that it is for the TLS support? thanks

Reply:
This change is unrelated to TLS. Still, I updated the diff to match https://github.com/paritytech/cachepot/pull/98.